tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -221,7 +221,7 @@ setup_agent_init()
 	[ -z "$init_bin" ] && die "need init bin path"
 
 	info "Install $agent_bin as init process"
-	mv -f "${agent_bin}" ${init_bin}
+	cp -f "${agent_bin}" ${init_bin}
```

Comment:
Doesn't this lead to a larger image size? Because now we'd have the agent binary twice, right?

Reply:
nop, agent_bin is a bind mount from the host and does not add to the guest rootfs size.